api/types/swarm/common.go

```diff
@@ -17,7 +17,7 @@ type Meta struct {
 // Annotations represents how to describe an object.
 type Annotations struct {
 	Name   string            `json:",omitempty"`
-	Labels map[string]string `json:",omitempty"`
+	Labels map[string]string `json:"Labels"`
```

Comment:
Thanks for confirming that this makes a difference.

I honestly don't know the right solution. It doesn't feel intuitively right to me to force the field to be marshalled in all cases so that the client avoids outputting `null` for a specific query. My sense is that it would be better for the client to make sure the map is initialized after unmarshalling, but I'm not sure how we handle these cases in general. I'll step back and let other maintainers offer opinions.